src/System.Private.ServiceModel/tests/Scenarios/Infrastructure/SetupValidationTests.cs

```diff
@@ -5,21 +5,37 @@
 
 using Infrastructure.Common;
 using System;
+using System.Security.Cryptography.X509Certificates;
 using Xunit;
 
 public class SetupValidationTests : ConditionalWcfTest
 {
+#if FULLXUNIT_NOTSUPPORTED
```

Comment:
Reviewers: I had forgotten to make these run on NET Native so am adding that support now.